Gemfile

```diff
@@ -15,3 +15,4 @@ gem 'packaging', '~> 0.4', :github => 'puppetlabs/packaging'
 gem 'rake'
 gem 'json'
 gem 'rubocop', "~> 0.34.2"
+gem 'rspec'
```

Comment:
I think this (and the line above) should be in something like a `group :development, :test do` block, since it's not needed at runtime?